### PR TITLE
add general rac_aspace description to README

### LINKED ISSUE #71: Add Description section to ReadMe
**Is your feature request related to a problem? Please describe.**
The "Description" section of the ReadMe currently has placeholder text.

**Describe the solution you'd like**
A few sentences describing the library (including what is in and out of scope) should be added to the Description section.

### PR DESCRIPTION
**Pull request by kemartin2015**
## Changes
*Adds overall package description to README. Fixes #71*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # rac_aspace
 
-*Write a brief description of the project, explaining what it does in a couple of sentences.*
+`rac_aspace` is a Python package of common functions based on the Rockefeller Archive Center’s previous coding efforts with the ArchivesSpace API . The goal of the package is to improve code maintainability and facilitate code reuse across our organization. The package implements consistent linting and unit tests and builds upon existing Python packages such as ArchiveSnake. The built in functionality includes data helpers, serialization, and deleting and saving in ArchiveSpace (see descriptions below).
```
Comment (helrond) [inline README.md]:
```suggestion
`rac_aspace` is a Python package which is intended to supplement [ArchivesSnake](https://github.com/archivesspace-labs/ArchivesSnake) by providing common functions based on the Rockefeller Archive Center’s previous coding efforts with the ArchivesSpace API. The goal of the package is to improve code maintainability and facilitate code reuse across our organization. The built in functionality includes data helpers, serialization, and deleting and saving in ArchiveSpace (see descriptions below).
```

**Review by helrond — COMMENTED** [review]
A couple of suggested changes to make things more concise.

**Comment by helrond** [comment]
@kemartin2015 I just realized this package _doesn't_ actually include saving or deleting data in AS, so I think we need to remove that last clause from the last sentence.

**Comment by p-galligan** [comment]
Looks good except for the saving/deleting part mentioned.

**Review by p-galligan — APPROVED** [review]

**Review by helrond — APPROVED** [review]

**Pull request merged**